lansa-vmss-windows-autoscale-sql-database/azuredeploy.json

```diff
@@ -0,0 +1,829 @@
+      "[concat(uniqueString(concat(resourceGroup().id, variables('newStorageAccountSuffix'), '4')))]"
+    ],
+    "saCount": "[length(variables('uniqueStringArray'))]",
+    "vhdContainerName": "[concat(variables('namingInfix'), 'vhd')]",
+    "osDiskName": "[concat(variables('namingInfix'), 'osdisk')]",
+    "addressPrefix": "10.0.0.0/16",
+    "subnetPrefix": "10.0.0.0/24",
+    "virtualNetworkName": "[concat(variables('namingInfix'), 'vnet')]",
+    "publicIPAddressName": "[concat(variables('namingInfix'), 'pip')]",
+    "dbpublicIPAddressName": "[concat(variables('namingInfix'), 'dbpip')]",
+    "subnetName": "[concat(variables('namingInfix'), 'subnet')]",
+    "loadBalancerName": "[concat(variables('namingInfix'), 'lb')]",
+    "dbloadBalancerName": "[concat(variables('namingInfix'), 'dblb')]",
+    "publicIPAddressID": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIPAddressName'))]",
+    "dbpublicIPAddressID": "[resourceId('Microsoft.Network/publicIPAddresses',variables('dbpublicIPAddressName'))]",
+    "lbID": "[resourceId('Microsoft.Network/loadBalancers',variables('loadBalancerName'))]",
+    "dblbID": "[resourceId('Microsoft.Network/loadBalancers',variables('dbloadBalancerName'))]",
+    "natPoolName": "[concat(variables('namingInfix'), 'natpool')]",
+    "dbnatPoolName": "[concat(variables('namingInfix'), 'dbnatpool')]",
+    "bePoolName": "[concat(variables('namingInfix'), 'bepool')]",
+    "dbbePoolName": "[concat(variables('namingInfix'), 'dbbepool')]",
+    "dbvmSku": "Standard_A1",
+    "natStartPort": 50000,
+    "natEndPort": 50119,
+    "natBackendPort": 3389,
+    "nicName": "[concat(variables('namingInfix'), 'nic')]",
+    "ipConfigName": "[concat(variables('namingInfix'), 'ipconfig')]",
+    "frontEndIPConfigID": "[concat(variables('lbID'),'/frontendIPConfigurations/loadBalancerFrontEnd')]",
+    "imagePublisher": "lansa",
+    "imageSku": "lansa-scalable-license-14-1",
+    "osType": {
+      "publisher": "[variables('imagePublisher')]",
+      "offer": "[parameters('imageOffer')]",
+      "sku": "[variables('imageSku')]",
+      "version": "latest"
+    },
+    "imageReference": "[variables('osType')]",
+    "diagnosticsStorageAccountName": "[concat(uniqueString(concat(resourceGroup().id, variables('newStorageAccountSuffix'))), variables('newStorageAccountSuffix') )]",
+    "diagnosticsStorageAccountResourceGroup": "[resourceGroup().name]",
+    "accountid": "[concat('/subscriptions/',subscription().subscriptionId,'/resourceGroups/',variables('diagnosticsStorageAccountResourceGroup'),'/providers/','Microsoft.Storage/storageAccounts/', variables('diagnosticsStorageAccountName'))]",
+    "sqlserverName": "[concat(variables('namingInfix'), 'sqlserver')]",
+    "gitRepo": "https://raw.githubusercontent.com/robe070/cookbooks/",
+    "gitRefreshName": "git-pull.ps1",
+    "gitRefreshUri": "[concat(variables('gitRepo'), parameters('gitBranch'), '/scripts/', variables('gitRefreshName'))]"
```

Comment:
The git repo is shared between all our deployment environments - AWS, Azure, Google and on-premise. We need to keep them in sync with the Build environment, not with the deployment environment. So we would rather leave it as it is.